src/commands/check.rs

```diff
@@ -319,14 +305,29 @@ pub fn check<'a>(
     };
 
     c.shell().status("Checking", "Packages")?;
+
+
+
```

Comment:
It'd be great to at least add a terse comment explaining why we can't calculate all of that beforehand, as we did before